### PR TITLE
Create s3 bucket and dynamodb table for terraform remote state

### PR DESCRIPTION
**Pull request by veziak**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -71,11 +72,6 @@
       "@oclif/plugin-update",
       "@oclif/plugin-warn-if-update-available"
     ],
-    "update": {
```
Comment (Spartakovic) [inline package.json:74]: why are you removing all these?

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "dgctl",
-  "version": "0.0.12",
+  "version": "0.0.8b",
```
Comment (Spartakovic) [inline package.json:3]: better set version to something actual

File: src/utils/remote-state.ts [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+  const bucketParams = {
+    Bucket: bucketName
+  };
+
+  let bucketExists = false;
+
+  const bucketsList = await s3Client.send(new ListBucketsCommand({}));
+  if (bucketsList && bucketsList?.Buckets) {
+    for (const element of bucketsList?.Buckets) {
+      if (element.Name == bucketName) bucketExists = true
```
Comment (Spartakovic) [inline src/utils/remote-state.ts:25]: you can just log and return here instead of using a flag

File: src/utils/remote-state.ts [hunk 4]
```diff
@@ -0,0 +1,154 @@
...
+import {
+  CreateBucketCommand,
+  ListBucketsCommand,
+  PutBucketEncryptionCommand,
+  PutBucketVersioningCommand, PutPublicAccessBlockCommand,
+  S3Client,
+} from "@aws-sdk/client-s3";
+
+
+const ddbClient = new DynamoDB({region: "us-east-1"});
```
Comment (Spartakovic) [inline src/utils/remote-state.ts:11]: it probably needs to ask for aws keys/profile? like we do in other commands

File: src/utils/remote-state.ts [hunk 5]
```diff
@@ -0,0 +1,154 @@
...
+}
+
+const createDynamoDB = async (ddbClient: DynamoDB, ddbTableName: string) => {
+
+  let ddbTableExists = false;
+
+  const data = await ddbClient.listTables({})
+  if (data && data?.TableNames ) {
+    for (const tableName of data?.TableNames) {
+      if (tableName == ddbTableName) ddbTableExists = true
```
Comment (Spartakovic) [inline src/utils/remote-state.ts:113]: same here

File: package.json (hunk 2 above)
Comment (veziak) [reply package.json:3]: oops, pushed it by mistake

File: package.json (hunk 1 above)
Comment (veziak) [reply package.json:74]: I used 'yarn install' to add dynamodb package, not sure why it changed some other things

File: src/utils/remote-state.ts (hunk 4 above)
Comment (veziak) [reply src/utils/remote-state.ts]: I call it just before we run 'terraform init' , at that point user set profile already I think

File: package.json [hunk 6]
```diff
@@ -98,12 +94,6 @@
       },
       "login": {
         "description": "Commands to login using Digger API key"
-      },
-      "eject": {
```
Comment (MatasJonys) [inline package.json:102]: This seems like a merge fail, since these are hand written and describe dgctl commands for the help seciton

Comment (veziak) [reply package.json]: updated it

**Pull request merged**